fv/ipip_test.go

```diff
@@ -57,7 +57,7 @@ var _ = infrastructure.DatastoreDescribe("IPIP topology before adding host IPs t
 		felixes, client = infrastructure.StartNNodeTopology(2, infrastructure.DefaultTopologyOptions(), infra)
 
 		// Install a default profile that allows all ingress and egress, in the absence of any Policy.
-		infra.AddDefaultAllow()
+		_ = infra.AddDefaultAllow()
```

Comment:
Don't think you need to add `_ = ` everywhere, does the linter complain?  Normally, I'd only do this if ignoring an error.

Reply:
Thanks, for some reason I thought that would have been a lint error.